icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -201,6 +201,16 @@ ant copy-cldr-testdata
 # and add the following line to the beginning of these two files
 # # File copied from cldr common/testData/localeIdentifiers/localeCanonicalization.txt
 
+# 4e. For the time being, manually re-add the lstm entries in data/brkitr/root.txt
+
+open $ICU4C_DIR/source/data/brkitr/root.txt 
+
+# paste the following block at the end, after the dictionaries block>
+    lstm{
+        Thai{"Thai_graphclust_model4_heavy.res"}
+        Mymr{"Burmese_graphclust_model5_heavy.res"}
+    }
+
```

Comment:
Presumably landed in https://unicode-org.atlassian.net/browse/ICU-21569 — is there a followon ticket for automating this?

Reply:
Well, automating this would involve putting the relevant lines in data/xml/brkitr/root.xml, and I started doing that as part of CLDR 40 integrations. The problem is that then creates a dependency that requires adding the data to the ICU4J jars, and the desire was not to do that. The data is currently just for testing so it can be enabled with a build flag, but for that to work these lines have to be present without going through the usual mechanism for adding support for new brkitr resources.